README.rst

```diff
@@ -3,45 +3,49 @@
   and fighting configuration battles.
-* **Multiple Environment Support**: The ADB works on Windows, Linux and Mac
-  OS X. The ADB supports several orchestrators (OpenShift, Kubernetes,
-  Mesos, and plain Docker). The ADB is language independent and supports
+* **Multiple Environment Support**: ADB works on Windows, Linux and Mac
+  OS X. ADB supports several orchestrators (OpenShift, Kubernetes,
+  Mesos, and plain Docker). ADB is language independent and supports
   multiple developer models (IDE, CLI, SSH containment).
-* **Production-Grade**: The components of the ADB are configured to behave
+* **Production-Grade**: The components of ADB are configured to behave
   just as they will in production. Containers promise seamless delivery,
   but only if you test them in the right environment. This is that
   environment.
-* **Self-Contained**: The ADB is ready to go once installed. It comes
+* **Self-Contained**: ADB is ready to go once installed. It comes
   prepackaged with the most common components ready, in case they
   are needed.
-* **Open Source**: The ADB leverages existing tools and technologies wherever possible to avoid pushing a developer into an environment that won't be supportable in production or that is tied to a single vendor.
+* **Open Source**: ADB leverages existing tools and technologies wherever
+  possible to avoid pushing a developer into an environment that won't be supportable
+  in production or that is tied to a single vendor.
   This also means it benefits from the stability of existing projects
   instead of reinventing the wheel.
 
-The ADB is a virtual machine that is executed with Vagrant and some Vagrant plugins.
+ADB is a virtual machine that is executed with Vagrant and some Vagrant plugins.
 
-When would the ADB typically be used?
+When would ADB typically be used?
 ===============================================================
 
-A developer begins using the ADB, in most cases, once they have a
+A developer begins using ADB, in most cases, once they have a
 working application that has been decomposed into micro-services.
 They then follow this general outline of steps:
 
-1. Consider ways to divide your application into its component services or micro-services. For standard pieces, such as web servers, consider using pre-built containers from trusted sources. For truly unique pieces, build a custom container.
-2. Confirm the application works in the ADB by manually launching the
+1. Consider ways to divide your application into its component services or
+   micro-services. For standard pieces, such as web servers, consider using
+   pre-built containers from trusted sources. For truly unique pieces, build a custom container.
+2. Confirm the application works in ADB by manually launching the
    application's container components or by using the instance of
    OpenShift in the container to launch the application.
 3. Build a Nulecule description of the application or complete the
```

Comment:
This should probably read:

3. Build orchestration configurations that provide scaling and other required features.